python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -369,12 +373,16 @@ def __call__(self, args):
             self.log.MDC("LABEL", str([ref.dataId for ref in dataRef if hasattr(ref, "dataId")]))
         task = self.makeTask(args=args)
         result = None  # in case the task fails
+        exitStatus = 0 # exit status for the shell
```

Comment:
Picky: code standards say two spaces between the functional end of the line and the comment.